docs/sources/panels/field-options/filter-table-columns.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "Filter table columns"
+keywords = ["grafana", "table options", "documentation", "format tables", "table filter", "filter columns"]
+type = "docs"
+weight = 600
++++
+
+# Filter table columns
+
+> **Note:** This feature is available in Grafana 7.2+.
+
+If you turn on the **Column filter** in [Table options](table-field-options.md), then you can filter table options.
+
+## Turn on column filtering
+
+1. In Grafana, navigate to the dashboard with the table with the columns that you want to filter.
+2. On the table panel you want to filter, [open the panel editor]({{< relref "../panel-editor.md#open-the-panel-editor" >}}).
+3. Click the **Field** tab.
+4. In Table options, turn on the **Column filter** option.
+
+A filter icon appears next to each column title.
+
+{{< docs-imagebox img="/img/docs/tables/column-filter-with-icon.png" max-width="500px" caption="Column filtering turned on" class="docs-image--no-shadow" >}}
```

Comment:
There is just one thing, somewhere out there there are users that were using the old table panel from Grafana <7. When they upgrade to 7 I think the Table panel is still the old one so they would need to actively choose the "new" Table panel to get this feature. Not sure this should be mentioned here but I can see that this can become confusing if your still using the "old" Table panel.

Reply:
I have not seen anything regarding choosing a table type. @dprokop, does this ring a bell?

Reply:
@oddlittlebird you can't choose the old table panel from the visualization options but if you are using a dashboard that was created in a previous version of Grafana then it is not automatically upgraded to the new table panel. An example on play: https://play.grafana.org/d/000000059/influxdb-table?editPanel=6&orgId=1 If you go to the visualization options and choose Table then you will see that the panel changes. I agree with Hugo, it is confusing and non-obvious.

Reply:
...omg, I see what you mean. I clicked the Table visualization and I suddenly lost a bunch of options. I'll try to figure out how to explain that.

Reply:
Updated. Please take a look.